Content.Shared/Damage/ForceSay/DamageForceSayComponent.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Content.Shared.Damage.Prototypes;
+using Content.Shared.FixedPoint;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
+
+namespace Content.Shared.Damage.ForceSay;
+
+/// <summary>
+/// This is used for forcing clients to send messages with a suffix attached (like -GLORF) when taking large amounts
+/// of damage, or things like entering crit or being stunned.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class DamageForceSayComponent : Component
+{
+    /// <summary>
+    ///     The localization string that the message & suffix will be passed into
+    /// </summary>
+    [DataField]
+    public string ForceSayMessageWrap = "damage-force-say-message-wrap";
+
+    /// <summary>
+    ///     Same as <see cref="ForceSayMessageWrap"/> but for cases where no suffix is used,
+    ///     such as when going into crit.
+    /// </summary>
+    [DataField]
+    public string ForceSayMessageWrapNoSuffix = "damage-force-say-message-wrap-no-suffix";
+
+    /// <summary>
+    ///     The fluent string prefix to use when picking a random suffix
+    /// </summary>
+    [DataField]
+    public string ForceSayStringPrefix = "damage-force-say-";
```

Comment:
These can be changed to use the LocId type with the latest engine update and they will be automatically validated by the linter.

(Might have to fix imports)

```suggestion
    [DataField]
    public LocId ForceSayMessageWrap = "damage-force-say-message-wrap";

    /// <summary>
    ///     Same as <see cref="ForceSayMessageWrap"/> but for cases where no suffix is used,
    ///     such as when going into crit.
    /// </summary>
    [DataField]
    public LocId ForceSayMessageWrapNoSuffix = "damage-force-say-message-wrap-no-suffix";

    /// <summary>
    ///     The fluent string prefix to use when picking a random suffix
    /// </summary>
    [DataField]
    public LocId ForceSayStringPrefix = "damage-force-say-";
```

Reply:
wtf our engine is based

Reply:
also the prefix one cant be linted but ill do the others